modules/traf/files/buildtest/stage-traf.sh

```diff
@@ -113,11 +113,11 @@ cd $workspace
 
 # installer
 install=$(ls ./trafodion/install/installer*gz ./trafodion/distribution/installer*gz)
-cp $install ./$DestDir/installer-$BLD.tar.gz  || exit 2
+cp $install ./$DestDir/apache-trafodion-installer-$BLD-incubating-bin.tar.gz  || exit 2
```

Comment:
That adds a few complications. The script would have to add code to pick up the env.sh from the build tree, plus it would have to modify content of that variable to lower case and replace spaces with hyphens.  I think sticking with hard-coding is better for now.